### PR TITLE
Changed to ECBP

### PR DESCRIPTION
**Pull request by TokenHash**
Changed file name from `erc.html` to `ecbp.html`

Changed title from `ERC` to `ECBP`.

Changed string `where:"category","ERC" %}` to `where:"category","ECBP" %}`

### CODE REVIEW

**Comment by sorpaas** [comment]
Would appreciate it if we can move #116, #115, #114 to this PR. Those needs to be merged together, otherwise the site generation will break.

**Comment by TokenHash** [comment]
If Everything went right I think I merged #119 + #117 + #116 + #115 + #114 all into #118.

I don't know why the PRs are still individually listed in the PR tab, but maybe the have to be closed.

All should be inside this one which is #118>

Thanks @BelfordZ for explaining how to merge the PRs! (I hope I did it right!)

**Comment by TokenHash** [comment]
I closed  #119 + #117 + #116 + #115 + #114.

File: index.html [hunk 1]
```diff
@@ -19,10 +19,13 @@ <h3>Standard Track ({{site.specs|where:"lang","en"|where:"type","Standards Track
 <h4><a href="{{"core"|relative_url}}">Core</a> ({{site.specs|where:"lang","en"|where:"type","Standards Track"|where:"category","Core"|size}})</h4>
 <p>Improvements requiring a consensus fork, as well as changes that are not necessarily consensus critical but may be relevant to “core dev” discussions.</p>
 
+<h4><a href="{{"networking"|relative_url}}">Networking</a> ({{site.specs|where:"lang","en"|where:"type","Standards Track"|where:"category","Networking"|size}})</h4>
```
Comment (sorpaas) [inline index.html]: The file “networking” does not exist. I think we need to create it (content should be similar to other categories files such as “ecbp”).

Comment (TokenHash) [reply index.html]: Good catch. I'm going to try to do the commit into this PR #118.

**Pull request merged**